tutorials/introductory/quilt_layout.py

```diff
@@ -0,0 +1,252 @@
+"""
+==============
+Patchwork Axes
+==============
+
+This is inspired by the `patchwork
+<https://github.com/thomasp85/patchwork>`__ library for R.  While we
+do not implement the operator overloading style, we do provide a
+pythonic API for specifying (nested) Axes layouts.  If you need to do
+something that is not clearly expressed using this method, please see
+:ref:`sphx_glr_tutorials_intermediate_gridspec.py`.
+
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def label_axes(ax_dict):
+    """
+    Helper to set the title on the Axes.
+
+    Parameters
+    ----------
+    ax_dict : Dict[str, Axes]
+        Mapping between the title / label and the Axes.  Set the left
+        title to be equal to the key.
+    """
+    for k, ax in ax_dict.items():
+        ax.set_title(k, loc="left")
+
+
+###############################################################################
+# Express String specification
+# ============================
+#
+# Using single leters we can "draw" the axes we want.  The following
+
+
+layout = """
+AAC
+BBC
+"""
+
+###############################################################################
+# will give us 3 axes.  Two will be stacked vertically in the left and
+# one will span the vertical space on the right.  The Axes on the left
+# will be twice as wide as the Axes on the right and the Axes on the
+# right will be twice as tall as Axes on the left.  But, as they say
+# a picture is worth a thousand words:
```

Comment:
First, this fails with constrained_layout, though not with #17494.

I'd be careful claiming the widths are double  With #17494 they indeed are, but without CL they are not.   However, note that if you do 
```
""""
AAC
BDE
""""
```
then A will *not* be twice as wide as C, but will be wider, and aligned with B and D.

Reply:
>But, as they say a picture is worth a thousand words

don't think this line adds anything to the tutorial